recipe/meta.yaml

```diff
@@ -3,7 +3,7 @@ package:
   version: 2.0.0
```

Comment:
As this is a pinning change, it should change the third version number instead. So, 2.0.1